readinglist/resource.py

```diff
@@ -130,7 +130,11 @@ def _extract_filters(self, queryparams):
             if param in self.known_fields:
                 filters.append((param, value, '=='))
             if param == '_since':
-                filters.append((self.modified_field, value, '>='))
+                if isinstance(value, six.integer_types + (float,)):
+                    filters.append((self.modified_field, value, '>='))
```

Comment:
I would have a set of constants for stuff like '>=' and '=='.  I have not checked but maybe Python lib has them already